app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(other: Herbivore) -> None:
```

Comment:
Fix annotations. If your function takes only Herbivore instances you won't need the check in the next line

Reply:
@LiudmylaKulinchenko .Thanks, i updated my code, but it is not a strict annotation, isnt it more understandable to write Herbivore? For example if other programmer will look at that method, he could think that any animal could be bited